### ISSUE TITLE
Caps rule no longer working?

### ISSUE THREAD

**Issue by mrcoffee1026** [kangarko/ChatControl-Pro#650]
Seems to act as if that's just set to "Enabled: false" now.  I reloaded chc a bunch of times though getting formatting set up so it may have to do with that and not the plugin itself, but I have noticed a prevalence of caps in use on the server for days which suggests this fails pretty often if there's a workaround for the problem...

Another strange one I got is that a user reported getting notification on books being written... strange as they did NOT have the chatcontrol.command.spy permission, and I'm not sure how else they would be getting those messages, any ideas?

**Comment by kangarko** [comment]
Strange, the anticaps from settings.yml works for me. Please post me your settings.yml without MySQL part and an example uppercase text to reproduce.

![a](https://i.imgur.com/LWnq72w.png) // all was in caps

For the second issue, the permission chatcontrol.spy.autoenable automatically enables spy on join, ensure they don't have it. The spy mode is enabled only in three places - via /spy command, on join, and from MySQL when previously enabled using the first two methods. 

Please check your MySQL database for the spy mode (Spying row) being enabled where it shouldn't and report. Else, ask out the user if he can recall getting the 'You are now spying...' message on join or whether he did run /spy and it worked for him. The reason I am asking you for testing is that I was not able to reproduce this either, but I had another user reporting a player seeing the spy chat, which is /spy related, so I believe the error is on my side and thus I need to track it down soonest possible. Thanks.

**Comment by mrcoffee1026** [comment]
When I restarted the server, his ability to spy the books went away.  I tried making another user with identical perms and got nothing out of that one, it seems to just be some freak thing.  I am certain he was never assigned the chatcontrol.spy.autoenable permission... I did do at least a dozen /chc reloads this last time and maybe wires got crossed along the way while that was going on?  Same seems to be true for the caps thing i just tested with an ALT and it auto lower-cased and gave the appropriate warning... prior to that I had a couple people report rather bizarre scenarios where short caps instances were called out... longer ones were not.  I think it may be entirely /chc reload related so there's probably not much that can be done.  Thanks for looking into this for me though.
